### ISSUE TITLE
ConfigurationExecutionError: <class 'zope.component.interfaces.ComponentLookupError'>: (<InterfaceClass zope.security.interfaces.IPermission>, 'plone.app.controlpanel.Site')

### ISSUE THREAD

**Issue by macagua** [collective/collective.sitelogo#2]
Hi @thet 

I am trying to test the collective.sitelogo product 1.0.1 (from PyPI) on a Plone 4.3.6 version with the following buildout configuration:

```
[buildout]
...
[instance]
...
eggs =
    collective.sitelogo
zcml =
    collective.sitelogo
```

And when I execute my Zope instance in foreground mode, show the following error:

```
2015-10-05 18:54:10 DEBUG config include /home/macagua/.buildout/eggs/plone.z3cform-0.8.0-py2.7.egg/plone/z3cform/overrides.zcml
---------------------------------------------------------------------------
ConfigurationExecutionError               Traceback (most recent call last)
/home/macagua/.buildout/eggs/Zope2-2.13.22-py2.7.egg/Zope2/Startup/run.py in <module>()
    347         del _options, _args
    348         with open(__file__, 'U') as __file__f:
--> 349             exec(compile(__file__f.read(), __file__, "exec"))
    350 
    351 if _interactive:

/home/macagua/.buildout/eggs/Zope2-2.13.22-py2.7.egg/Zope2/Startup/run.py in <module>()
     74 
     75 if __name__ == '__main__':
---> 76     run()
     77 

/home/macagua/.buildout/eggs/Zope2-2.13.22-py2.7.egg/Zope2/Startup/run.py in run()
     20     starter.setConfiguration(opts.configroot)
     21     try:
---> 22         starter.prepare()
     23     except:
     24         starter.shutdown()

/home/macagua/.buildout/eggs/Zope2-2.13.22-py2.7.egg/Zope2/Startup/__init__.pyc in prepare(self)
     84         self.makePidFile()
     85         self.setupInterpreter()
---> 86         self.startZope()
     87         self.serverListen()
     88         from App.config import getConfiguration

/home/macagua/.buildout/eggs/Zope2-2.13.22-py2.7.egg/Zope2/Startup/__init__.pyc in startZope(self)
    260         # Import Zope
    261         import Zope2
--> 262         Zope2.startup()
    263 
    264     def makeLockFile(self):

/home/macagua/.buildout/eggs/Zope2-2.13.22-py2.7.egg/Zope2/__init__.pyc in startup()
     45     _configure()
     46     from Zope2.App.startup import startup as _startup
---> 47     _startup()
     48 
     49 def app(*args, **kw):

/home/macagua/.buildout/eggs/Zope2-2.13.22-py2.7.egg/Zope2/App/startup.pyc in startup()
    116 
    117     # Set up the CA
--> 118     load_zcml()
    119 
    120     # Set up the "app" object that automagically opens

/home/macagua/.buildout/eggs/Zope2-2.13.22-py2.7.egg/Zope2/App/startup.pyc in load_zcml()
     50     # This hook is overriden by ZopeTestCase
     51     from .zcml import load_site
---> 52     load_site()
     53 
     54     # Set up Zope2 specific vocabulary registry

/home/macagua/.buildout/eggs/Zope2-2.13.22-py2.7.egg/Zope2/App/zcml.pyc in load_site(force)
     44 
     45     global _context
---> 46     _context = xmlconfig.file(site_zcml)
     47 
     48 

/home/macagua/.buildout/eggs/zope.configuration-3.8.0-py2.7.egg/zope/configuration/xmlconfig.pyc in file(name, package, context, execute)
    646     include(context, name, package)
    647     if execute:
--> 648         context.execute_actions()
    649 
    650     return context

/home/macagua/.buildout/eggs/zope.configuration-3.8.0-py2.7.egg/zope/configuration/config.pyc in execute_actions(self, clear, testing)
    689                 info = action['info']
    690                 try:
--> 691                     callable(*args, **kw)
    692                 except (KeyboardInterrupt, SystemExit): # pragma: no cover
    693                     raise

/home/macagua/.buildout/eggs/AccessControl-3.0.8-py2.7-linux-i686.egg/AccessControl/security.pyc in protectClass(klass, permission_id)
    163         security.declareObjectPrivate()
    164     else:
--> 165         permission = getUtility(IPermission, name=permission_id)
    166         # Zope 2 uses string, not unicode yet
    167         perm = str(permission.title)

/home/macagua/.buildout/eggs/zope.component-3.9.5-py2.7.egg/zope/component/_api.pyc in getUtility(interface, name, context)
    167     if utility is not None:
    168         return utility
--> 169     raise ComponentLookupError(interface, name)
    170 
    171 def queryUtility(interface, name='', default=None, context=None):

ConfigurationExecutionError: <class 'zope.component.interfaces.ComponentLookupError'>: (<InterfaceClass zope.security.interfaces.IPermission>, 'plone.app.controlpanel.Site')
  in:
  File "/home/macagua/.buildout/eggs/collective.sitelogo-1.0.1-py2.7.egg/collective/sitelogo/configure.zcml", line 33.2-39.8
    <browser:page
        name="sitelogo-controlpanel"
        for="Products.CMFPlone.interfaces.IPloneSiteRoot"
        class=".controlpanel.SiteControlPanel"
        layer="collective.sitelogo.interfaces.IThemeSpecific"
        permission="plone.app.controlpanel.Site"
        />
```

Any idea about this error?

**Comment by thet** [comment]
I've included persmissions.zcml from plone.app.controlpanel. I guess your issue is fixed now. Couldn't reproduce, though.
A new version is available: 1.0.2

**Comment by macagua** [comment]
@thet Done! 

Now it's working for me, thank you!
